cics/cmci/reporting/README.md

```diff
@@ -46,7 +49,7 @@ The `cmci_*` modules have pre-requisites that need to be installed into the Pyth
 executes.  In this case, the `cmci_get` module will be executed on `localhost`, i.e. the Ansible control
 node.  The playbook demonstrates how you can ensure the pre-requisites are installed (wherever the module runs) before
 the `cmci_get` module is executed.  More information about the `cmci_*` module pre-requisites can be found in the
-[documentation](todo)
+[documentation](todo).
```

Comment:
Couple of todo links - assume you're waiting for the doc to be in the right place first.

Reply:
Yep, I was going to publish the doc before merging this